client/config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml

```diff
@@ -72,6 +72,11 @@ spec:
       name: Age
       type: date
     name: v1beta1
+    # This indicates the v1beta1 version of the custom resource is deprecated.
+    # API requests to this version receive a warning in the server response.
+    deprecated: true
```

Comment:
Just to make sure, this will **not** stop API server from serving, just matter of when users try to use the beta api, it will give a warning from API server, correct?

Reply:
Yes